plc4x/throwing-the-towel.adoc

```diff
@@ -0,0 +1,227 @@
+
+If there's nobody willing or able to do that, that's going to create facts a lot quicker than most people will expect.
+
+Even if money would definitely be well invested in supporting open-source projects in general.
+Especially those projects that don't have any form of commercial backing are probably most in need of funding, as they will not have the power to prepare for these legal changes.
+Some rules for some projects will most probably require them to execute external code audits.
+Which group of private people will be willing to invest the significant amounts of their private money in order to pay for something, just to be able to continue giving away something for free?
+I see no way around significant investments by entities relying on open-source but also legal bodies such as the European Union in order to keep the lights on in these projects.
+
+One thing people relying on open-source should generally consider: Using an open-source project in the EU which is hosted at an organisation which is considered an  `open-source steward` will reduce the amount of hoops they definitely will need to jump through in order to sell their products significantly once CRA and PLD are in place.
+The ASF is such a place.
+
+Using projects that are not maintained by an open-source steward might become impossible, as these would either need to apply to all rules or using them will become illegal if the company using it doesn't take care of this (That's a hell of a lot of rules).
+Apache PLC4X is, as far as I know, the only open source project for communication with various types of industrial equipment, which is also hosted by an organization considered an open-source steward.
+The only other project in this sector I know of would be the Eclipse Milo project, as the Eclipse foundation is also considered an open-source steward.
+Having a look at most other open-source projects in that field.
+Most are un-paid single person projects.
+I see no chance of being able to continue to use these in a CRA and PLD regulated world.
+NodeRED is going to be interesting, but I am not a lawyer and I can't say what's going to happen there.
+
+Implementing things themselves will require not only involve investing a lot of work, but also jumping through the complete list of hoops.
+
+So in general: Companies relying on open-source should generally consider funding the projects and the foundations where they are planning to continue consuming open-source products.
+I know the ASF is already preparing itself, and this comes with quite some costs for the foundation.
+I actually just helped approve the budget increases in the last board meeting.
+If these additional costs make the Foundations run at a deficit, these safe harbours might also go away, leaving the commercial a world full of hoop-jumpers.
```

Comment:
If -> When (da zeitlich möglich in der Zukunft)
go away -> disappear ?

"leaving the commercial a world full of hoop-jumpers"
The commercial what?
leaving the commercial world as one full of hoop-jumpers.
In meinem Kopf hab ich gerade ne menge Hoolahoops 😄